wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -2819,7 +2819,7 @@ function ( $table ) {
 					} elseif ( isset( $table['TABLE_NAME'] ) ) {
 						$table_name = $table['TABLE_NAME'];
 					}
-					return $table_name && ! array_key_exists( $table_name, $this->sqlite_system_tables );
+					return ! array_key_exists( $table_name, $this->sqlite_system_tables );
```

Comment:
Shouldn't we still be checking for `$table_name != ''` otherwise it would return true for the case when code didn't reach any of the if-elses, which I don't think is desirable?

```suggestion
					return $table_name != '' && ! array_key_exists( $table_name, $this->sqlite_system_tables );
```

Reply:
If we removed tables without names, you wouldn't be able to use an alias for the name column.
We allow empty table names because if the query uses an alias for the table name (`SELECT name as alias_name ...`) we don't have a way of detecting the name in the result. 

I updated the inline documentation to clarify this [70c0ea9](https://github.com/Automattic/sqlite-database-integration/pull/3/commits/70c0ea9a4a9fdb9f46ef3da340550051e99efb0d)

Also, this test covers that scenario https://github.com/Automattic/sqlite-database-integration/pull/3/files#diff-8e9f5a09318f3c475e1a648142b2ec981e5aa29bfe06755e0c510e3f37a5e38bR135-R136

Reply:
Thanks for explaining! Just wanted to bring attention to that in case it would have produced an undesirable result.